### ISSUE TITLE
Create Server-Side Pool Candidate CSV Report Generator

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#9687]
## ✨ Feature

We want to start generating the Pool Candidate CSV reports on the server.  Let's set up the view to do that.

## 🕵️ Details

This issue only deals with the CSV report itself.  How the client requests it and how it gets back to the client are out of scope for now.

## 🎨 Design File

The server-side report should match the current client-side report design.

## ✅ Acceptance Criteria

- [ ] It is possible to generate a CSV with one or more pool candidates on the server
  - [ ] A dev can trigger the request manually.  A regular user cannot.
  - [ ] No need to authenticate request yet
  - [ ] No need to save the file on the server, can just send back an HTTP response
- [ ] Report matches the report generated by the client

## 🛑 Blockers

Issues which must be completed before this one.

```[tasklist]
### Blocked By
```

**Comment by tristan-orourke** [comment]
If possible, let's generate excel files instead of CSV. That will allow us to produce richer files for our users later.

**Comment by tristan-orourke** [comment]
> If possible, let's generate excel files instead of CSV. That will allow us to produce richer files for our users later.

Nevermind, we'll keep it a simple CSV for now
